src/web/routes/application/check/get.js

```diff
@@ -1,16 +1,109 @@
-const pageContent = {
-  title: 'Check',
-  heading: 'Check'
+const { YES } = require('../are-you-pregnant/constants')
+
+const pageContent = ({ translate, claim }) => ({
+  title: translate('check.title'),
+  heading: translate('check.heading'),
+  sendApplicationHeader: translate('check.sendApplicationHeader'),
+  sendApplicationText: translate('check.sendApplicationText'),
+  buttonText: translate('buttons:acceptAndSend'),
+  checkRowData: buildCheckRowData(translate, claim)
+})
+
+const buildCheckRowData = (translate, claim) => {
+  return [
+    buildNameRow(translate, claim),
+    buildNinoRow(translate, claim),
+    buildDateOfBirthRow(translate, claim),
+    buildAreYouPregnantRow(translate, claim),
+    buildExpectedDeliveryDateRow(translate, claim),
+    buildAddressRow(translate, claim)
+  ].filter(row => row !== undefined)
+}
+
+const buildNameRow = (translate, claim) => {
+  return buildRowData(
+    translate('check.name'),
+    [claim.firstName, claim.lastName].join(' ')
+  )
+}
+
+const buildNinoRow = (translate, claim) => {
+  return buildRowData(
+    translate('check.nationalInsuranceNumber'),
+    claim.nino
+  )
+}
+
+const buildDateOfBirthRow = (translate, claim) => {
+  return buildRowData(
+    translate('check.dateOfBirth'),
+    [claim['dateOfBirth-day'], claim['dateOfBirth-month'], claim['dateOfBirth-year']].join(' ')
+  )
+}
+
+const buildAreYouPregnantRow = (translate, claim) => {
+  return buildRowData(
+    translate('check.areYouPregnant'),
+    capitaliseString(claim.areYouPregnant)
```

Comment:
This raises an interesting point about translations - we're not translating yes/no. Could we instead look up a translation for 'yes' and 'no'? Probably one for another pr...

Reply:
Raised as a subtask: HTBHF-472

Reply:
Yep, you're right, the latest commit translates it.